### DISCUSSION TITLE
Get nitendo dsi browser to work

### DISCUSSION THREAD

**Discussion by ghost** [rafaelvcaetano/melonDS-android#762] [Support]
Justafridge here got nitendo dsi browser to work
But i cant do it because it tells me to accept user agreement that does not load
So justafridge how do you get nitendo dsi browser working here?

**Comment by Damaj301damaj-lol** [comment]
You can't, dsi/ds has no internet.

**Comment by justafridge** [comment]
I was just looking at this repo after quite a while and noticed this discussion, huh.
Anyway, back to the topic of the discussion, the user agreement should work on Android if I recall correctly, if it doesn't, accept it on PC and transfer back the DSi NAND to your phone, that way the DSi Browser should just work, I just checked and I had previously set the Google DNS (8.8.8.8) in the DSi Settings

**Reply by justafridge** [reply]
Also, be sure to note that the DSi Browser will take quite a while before it actually connects to the internet, you'll get the green connection symbol basically instantly but you'll have to wait for the "Connecting to the internet..." pop-up to disappear
